publisher/presenter/html/handlers.go

```diff
@@ -78,6 +78,10 @@ func (h *handlers) ContainerIndex(c echo.Context) error {
 
 	pagesCount, containers, err := h.svc.ListContainersByPage(c.Request().Context(), namespace, page)
 	if err != nil {
+		log.Warnf("%s", err)
```

Comment:
Here it would be good to describe were it come from i.e. what method, etc. and another thing here is it probably should be an Info level since in some cases 404 is expected code if there're some logic behind.

Reply:
Deleted that log point, needed just for debugging